xml/admin_monitoring_alerting.xml

```diff
@@ -133,39 +97,19 @@
 
  <sect1 xml:id="prometheus">
    <title>&prometheus;</title>
-  <para>The exporter based configuration that can be passed through the pillar.
-    These groups map to exporters that provide data. The node exporter is
-    present on all nodes, &ceph; is exported by the &mgr; nodes, &prometheus;
-    and &grafana; is exported by the respective &prometheus; and &grafana; nodes.</para>
+  <para>
+    <!-- Add intro about prometheus
+  https://docs.ceph.com/docs/master/mgr/prometheus/ (see below for variable list)-->
+    The node exporter is present on all nodes. &ceph; data is exported by
```

Comment:
Maybe `should be present` or `It is highly recommended that the node-exporter runs on all nodes`. (This can be done using the `monitoring.yaml` configuration above. The service type `node-exporter` does exactly that.)

Reply:
When the node exporter does not run on all nodes (hosts), those hosts where the node-exporter is missing will not be monitored and host alerts (not directly related to ceph but the host) aren't fired for those hosts.

The [prometheus manager module](https://ceph.readthedocs.io/en/latest/mgr/prometheus/) is a module inside Ceph that extends Ceph's functionality. Its whole purpose is to read (meta) data from Ceph about its state and health and provides those in a format consumable (scrapeable) by prometheus.

High level overview would be that node-exporters is monitoring the hosts and the prometheus manager module is monitoring the cluster (where multiple services of the same and different kinds can run on a single host)